community/modules/scripts/ramble-setup/scripts/install_ramble_deps.yml

```diff
@@ -0,0 +1,54 @@
+# Copyright 2023 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+---
+
+- name: Install dependencies for ramble installation
+  become: yes
+  hosts: localhost
+  tasks:
+  - name: Install dependencies through system package manager
+    ansible.builtin.package:
+      name:
+      - python3-pip
+      - git
+  - name: Gather the package facts
+    ansible.builtin.package_facts:
+      manager: auto
+  - name: Install protobuf for old releases of Python
+    when: ansible_facts.packages["python3"][0].version is version("3.7", "<") and ansible_facts.packages["python3"][0].version is version("3.5", ">=")
+    ansible.builtin.pip:
+      name: protobuf
+      version: 3.19.4
+      executable: pip3
+  - name: Install google cloud storage
+    ansible.builtin.pip:
```

Comment:
It looks like these pip installs can be taken as a list. See [examples](https://docs.ansible.com/ansible/latest/collections/ansible/builtin/pip_module.html#examples). Let's convert to that format unless you have an objection, at least where there are not additional conditions needed to be met.

Reply:
I actually updated this to use the requirements.txt